Gemfile.jruby-2.3.lock.release

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ./logstash-core
   specs:
-    logstash-core (6.1.3-java)
```

Comment:
wow how did this happen? does a 6.1.4 work correctly without needs this to be a correct value?